improver/cli/__init__.py

```diff
@@ -332,6 +340,9 @@ def with_output(
         output (str, optional):
             Output file name. If not supplied, the output object will be
             printed instead.
+        tee (bool):
```

Comment:
I like having this option, but I'm not a fan of the name :-)
It's in reference to bash `tee`? My preference would be to call it something more transparent (but it is personal preference, so up to you)

Reply:
I must confess I find this a useful description - as it mirrors what shell tee does.. any thoughts for a clearer alternative?

Reply:
Because you know shell ;-)
But a lot of the devs for improver don't, which means this won't be as obvious to them, how about  something really clear, e.g. `pass_through_output`. 
Actually, being really picky, this doesn't behave like shell `tee`, if we specify `output` as `False` then `tee` actually does nothing

Reply:
I didn't originally know about the tee bash reference myself until I looked it up.
`pass_through_output` has my vote 👍

Reply:
Or perhaps `always_pass_through_output` might be more descriptive?

Reply:
yeah, any variation of pass through, or return output 👍 
would it want to be the default? we usually want to return the output even if writing out?

Reply:
> would it want to be the default? we usually want to return the output even if writing out?

I think that might make sense given that I *think* it would only be the end of the chains that wouldn't need passing their results on in memory.

Reply:
> > would it want to be the default? we usually want to return the output even if writing out?
> 
> I think that might make sense given that I _think_ it would only be the end of the chains that wouldn't need passing their results on in memory.

FYI, clize will by default print the return value if it's not `None`, which is why I haven't made it the default.